Makefile

```diff
@@ -108,6 +108,13 @@ test-pylint:
 test-pyright:
 	pyright .
 
+.PHONY: test-shellcheck
+test-shellcheck:
+	# shellcheck for shell scripts
+	git ls-files | xargs file -N | grep shell.script | cut -f1 -d: | xargs shellcheck
```

Comment:
Can you change `starbase` to use this for shellcheck too? It's much better than the current approach IMO

Reply:
Will do!